assets/airports/kpit.json

```diff
@@ -0,0 +1,4316 @@
+                "KPIT10R": ["LEMEW", "#300"],
+                "KPIT14": [],
+                "KPIT28L": ["BBLAK", "GOLDH", "JOEPA", "GUINZ", "CYDNY", "#100"],
+                "KPIT28C": ["BBLAK", "GOLDH", "JOEPA", "GUINZ", "CYDNY", "#100"],
+                "KPIT28R": ["BBLAK", "GOLDH", "JOEPA", "GUINZ", "CYDNY", "#100"],
+                "KPIT32": ["BBLAK", "GOLDH", "FLURY", "MLKIN", "CRSBY", "#145"]
+            },
+            "draw": [
+                ["FEWGA", "BBLAK", "GOLDH", "JOEPA", "GUINZ", "CYDNY"],
+                ["CKB*", "AIR", "WISKE", "FEWGA", "LEMEW"],
+                ["GOLDH", "FLURY", "MLKIN", "CRSBY"],
+                ["JPU*", "PYRAT", "AIR"],
+                ["CTW*", "AIR"],
+                ["HNN*", "JPU"]
+            ]
+        },
+        "HAYNZ4": {
+            "icao": "HAYNZ4",
+            "name": "Haynes Four",
+            "entryPoints": {
+                "CIP": ["CIP"],
+                "PSB": ["PSB"],
+                "SLT": ["SLT"]
+            },
+            "body": ["EARED", "GRACE", "HAYNZ"],
+            "rwy": {
+                "KPIT10L": ["ROOSA", "TERBL", "TOWEL", "#281"],
+                "KPIT10C": ["ROOSA", "TERBL", "TOWEL", "#281"],
+                "KPIT10R": ["ROOSA", "TERBL", "TOWEL", "#281"],
+                "KPIT14": [],
+                "KPIT28L": ["MYRON", "#125"],
+                "KPIT28C": ["MYRON", "#125"],
+                "KPIT28R": ["MYRON", "#125"],
+                "KPIT32": ["MYRON", "COPPE", "#165"]
+            },
+            "draw": [
+                ["CIP*", "EARED", "GRACE", "HAYNZ", "MYRON", "COPPE"],
+                ["HAYNZ", "ROOSA", "TERBL", "TOWEL"],
+                ["PSB*", "EARED"],
+                ["SLT*", "EARED"]
+            ]
+        },
+        "JESEY2": {
+            "icao": "JESEY2",
+            "name": "Jesse Two",
+            "entryPoints": {
+                "ACO": ["ACO"],
+                "BSV": ["BSV"],
+                "YNG": ["YNG"]
+            },
+            "body": ["CUTTA", "JESEY"],
+            "rwy": {
+                "KPIT10L": ["YINZZ", "#250"],
+                "KPIT10C": ["YINZZ", "#250"],
+                "KPIT10R": ["YINZZ", "#250"],
+                "KPIT14": [],
+                "KPIT28L": ["KRIKK", "SSOXX", "ROOSA", "#101"],
+                "KPIT28C": ["KRIKK", "SSOXX", "ROOSA", "#101"],
+                "KPIT28R": ["KRIKK", "SSOXX", "ROOSA", "#101"],
+                "KPIT32": ["KRIKK", "PROGY", "PRUIT", "#145"]
+            },
+            "draw": [
+                ["JESEY", "KRIKK", "SSOXX", "ROOSA"],
+                ["ACO*", "CUTTA", "JESEY", "YINZZ"],
+                ["KRIKK", "PROGY", "PRUIT"],
+                ["BSV*", "CUTTA"],
+                ["YNG*", "CUTTA"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "KPIT",
+            "destination": "",
+            "category": "departure",
+            "route": "KPIT.PIT2.AHTIY..BURGS",
```

Comment:
These and all spawn patterns will need to include runways, such that the entry/exit matches that which is specified in the procedure definition. This is true in spawn patterns and any time being issued a reroute. It has to do with the way routes are interpreted.